src/navigator/routes.js

```diff
@@ -1,5 +1,11 @@
 export const ROUTES = {
+  BottomTabNavigator: 'BottomTabNavigator',
   SignIn: 'SignIn',
   SignUp: 'SignUp',
   Dashboard: 'Dashboard',
+  Packages: 'Packages',
+  Profile: 'Profile',
+  'All Offers': 'All Offers',
+  'Created Offers': 'Created Offers',
+  'Accepted Offers': 'Accepted Offers',
```

Comment:
We can use key name `AllOffers` then we wont need quotes in key name and will be able to access it like `ROUTES.AllOffers`